javascript/javascript3/week1/GiphySearchApp/index.js

```diff
@@ -0,0 +1,25 @@
+//api_key=GBqVaF0BqJQ6cFmXKetzAu1SgA2HlFrA
```

Comment:
Nice variable namings and a nice function, there's not alot I would change here. Same thing as I commented on below about the definition of the API key, but that's probably the only thing here.

Reply:
super .. thanks